Documentation/SpatialAwareness/ConfiguringSpatialAwarenessMeshObserver.md

```diff
@@ -1,123 +1,122 @@
-# Configuring the Spatial Awareness Mesh Observer
+# Configuring Mesh Observers for Device
 
-For every Spatial Mesh Observer entry in the Spatial Awareness System profile, the Mesh Observer Profile must define the type implementation and the list of supported platform(s) to run this observer.
+SDFSDFSDF
+Explain here
 
-The default implementation type for [IMixedRealitySpatialAwarenessMeshObserver](xref:Microsoft.MixedReality.Toolkit.SpatialAwareness.IMixedRealitySpatialAwarenessMeshObserver) provided by the Mixed Reality Toolkit is the [WindowsMixedRealitySpatialMeshObserver](xref:Microsoft.MixedReality.Toolkit.WindowsMixedReality.SpatialAwareness.WindowsMixedRealitySpatialMeshObserver).
+All observers must extend the [IMixedRealitySpatialAwarenessMeshObserver](xref:Microsoft.MixedReality.Toolkit.SpatialAwareness.IMixedRealitySpatialAwarenessMeshObserver) interface. Furthermore, the default implementation provided by the Mixed Reality Toolkit is the [WindowsMixedRealitySpatialMeshObserver](xref:Microsoft.MixedReality.Toolkit.WindowsMixedReality.SpatialAwareness.WindowsMixedRealitySpatialMeshObserver) for the Windows Mixed Reality platform.
 
-![Mesh Observer General Settings](../../Documentation/Images/SpatialAwareness/SpatialAwarenessMeshObserverProfile_TypesPlatforms.png)
-
-The spatial awareness mesh observer profile also provides options for configuring:
-
-- [General Settings](#general-settings)
-- [Physics Settings](#physics-settings)
-- [Level of Detail Settings](#level-of-detail-settings)
-- [Display Settings](#display-settings)
-
-## General Settings
+## Profile Settings
 
-![Mesh Observer General Settings](../../Documentation/Images/SpatialAwareness/MeshObserverGeneralSettings.png)
+The following two items must be defined first when configuring a Spatial Mesh Observer profile for the [Spatial Awareness system](SpatialAwarenessGettingStarted.md).
 
-### Startup Behavior
+1. The concrete observer type implementation
+1. list of supported platform(s) to run this observer
 
-The startup behavior specifies the behavior of the observer when it is first instantiated. The options are Auto Start and Manual Start. The default value is Auto Start. When set to Manual Start, applications will need to call one of the following [IMixedRealitySpatialAwarenessSystem](xref:Microsoft.MixedReality.Toolkit.SpatialAwareness.IMixedRealitySpatialAwarenessSystem) methods:
+> [!NOTE]
```

Comment:
This duplicates the above.

Reply:
I don't understand? What is duplicated?

Reply:
Lines 6 and 16

Reply:
You must be looking at an oudated piece or github messed up, those lines are spaces